federator/src/lib/TransactionSender.ts

```diff
@@ -136,14 +140,21 @@ export class TransactionSender {
       r: 0,
       s: 0,
     };
-
+    
     if (await this.isRsk()) {
       delete rawTx.chainId;
       delete rawTx.r;
       delete rawTx.s;
     }
-    rawTx.gas = this.numberToHexString(await this.getGasLimit(rawTx));
 
+    const gasLimit = await this.getGasLimit(rawTx);
+
+    if (gasLimit === null) {
+      return null;
```

Comment:
Instead of asking for null gas we should fix the error that is being caused on calculating the gas